.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```

Comment:
please add anything .vscode related to .gitignore

Reply:
ok did refactor the gitignore